### ISSUE TITLE
Accessibility, Section 508 Compliance: Reports widget <svg>'s are not accessible

### ISSUE THREAD

**Issue by jdm4344** [Azure/api-management-developer-portal#1375]
## Bug description

Reports widget graph and map `<svg>` elements are missing accessibility attributes.

## Reproduction steps

N/A

## Expected behavior

`<svg>` elements have the necessary ARIA attributes to make them recognizable by accessibility software and either a descriptive `<title>` or `<desc>` as its first child element.

## Is your portal managed or self-hosted?

Managed 

## Environment

 - Operating system: MacOS 10.15.7 - Macbook Pro 15-inch, 2017 model
 - Browser: Google Chrome
 - Version: 91
 
## Additional context

<img width="1019" alt="Map - Attributes" src="https://user-images.githubusercontent.com/31706474/125687605-fbe31c3c-47bc-4c04-9cb8-a0a82509dd8d.png">

Related Section 508 test step: https://section508coordinators.github.io/TrustedTester/images.html#for-meaningful-images--check-111-meaningful-image-name

**Comment by christopherwoodland** [comment]
Thank you for fixing this issue!
